kura/org.eclipse.kura.core.configuration/src/main/java/org/eclipse/kura/core/configuration/ConfigurationServiceImpl.java

```diff
@@ -1164,6 +1180,11 @@ String getSnapshotsDirectory() {
 
     private File getSnapshotFile(long id) {
         String configDir = getSnapshotsDirectory();
+
+        if (configDir == null) {
```

Comment:
invalid parameter or something like this. But it's a private method so null will do